04-spatial-operations.Rmd

```diff
@@ -303,7 +304,7 @@ detach("package:tmap", unload = TRUE)
 
 The simplest useful method for spatial interpolation is *area weighted* spatial interpolation.
 This is implemented in `st_interpolate_aw()`, as demonstrated below:
-
+<!-- well, what you are doing here is weighting the inhabitants presumably on the basis of the area. The link between your congruent and incongruent example is somehow missing. One gets the impression, that the spatial interpolation takes care of incongruencies which is not the case. So I would suggest to clarify the weighting procedure.-->
```

Comment:
I would suggest to make clear that `st_interpolate_aw()` not sums up the inhabitants but weighs them by area. And yes, this is a simple approach and better than just to sum up the inhabitants but imagine your circle just intersects the desert parts of a country.